test/jason_encoder_test.exs

```diff
@@ -35,6 +35,6 @@ defmodule ExOpenAI.JasonEncoderTest do
     ]
 
     assert encode_and_return(msgs) ==
-             "[{\"content\":\"Hello!\",\"role\":\"user\"},{\"content\":\"What's up?\",\"role\":\"assistant\"},{\"content\":\"What ist the color of the sky?\",\"role\":\"user\"}]"
+             "[{\"role\":\"user\",\"content\":\"Hello!\"},{\"role\":\"assistant\",\"content\":\"What's up?\"},{\"role\":\"user\",\"content\":\"What ist the color of the sky?\"}]"
```

Comment:
I guess this test might be flaky
So to avoid these changes in order of parameters, it's possible to do something like this:
```
assert encode_and_return(msgs) =~ "{\"content\":\"Hello!\",\"role\":\"user\"}]"
assert encode_and_return(msgs) =~ "{\"role\":\"user\",\"content\":\"Hello!\"}"
assert encode_and_return(msgs) =~ "{\"role\":\"user\",\"content\":\"What ist the color of the sky?\"}"
```